.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea
 github.token
+.generated-go-semantic-release-changelog.md
```

Comment:
Is this something new? I have never seen this before on other repos we are using semantic release on

Reply:
Its because in the other repos, we are not doing goreleaser afterwards. The go semantic release creates these causing git to be dirty

Reply:
And unfortunately there is no --cleanup flag for go-semantic-release

Reply:
Do you mean we have explicitly configure semantic release to release this so goreleaser can use it?

Reply:
Not sure I understand, we replaced the node semantic release tool with go semantic release tool. Unfortunately go semantic release leaves generated stuff in the git root, which node didn’t. Hence the gitignore

Reply:
Ah different semantic release tool. Got it

Reply:
😀